application/common/models/Mfa.php

```diff
@@ -346,6 +361,7 @@ public static function create(int $userId, string $type, string $label = null, s
             $mfa = new Mfa();
             $mfa->user_id = $userId;
             $mfa->type = $type;
+            $mfa->recovery_email = $recoveryEmail;
```

Comment:
I don't fully understand: why do we need to save the recovery email address to the database? It seems like we could simply pass it to regInit (below, on new line 384), like we do with `rpOrigin`.

Feel free to fill me in on your thinking here, as I expect you have some meaningful reason for having done it this way. (Thanks)

Reply:
Good point, we made the create first, and for some reason I was thinking we'd need it for verification, but we don't. we can remove this.